### PR TITLE
Register Cxx

### PR DESCRIPTION
**Pull request by Keno**
A humble package to interface Julia with C++.

### CODE REVIEW

**Comment by tkelman** [comment]
License should acknowledge any code copied from base.

**Comment by Keno** [comment]
There isn't any.

**Comment by tkelman** [comment]
makefiles.

You also need a REQUIRE file in the package itself.

File: Cxx/versions/0.0.1/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+192a3fd1b7696ceb4e4ff565cbf5b9725c881cad
```
Comment (tkelman) [inline Cxx/versions/0.0.1/sha1]: this commit isn't pushed to a branch?

Comment (Keno) [inline Cxx/versions/0.0.1/sha1]: It's the one that added the REQUIRE file ;).

**Comment by Keno** [comment]
https://github.com/Keno/Cxx.jl/commit/f6bc213b335e3c87a6dd2a08a0846813a259abfe for LICENSE adjustments, though I should note that I consider the parts of the Makefiles fair use, and nothing in that commit should be construed to the contrary.

**Pull request merged**